pkgs/by-name/ta/tailscale/package.nix

```diff
@@ -77,6 +79,14 @@ buildGo123Module {
       moveToOutput "bin/derper" "$derper"
       moveToOutput "bin/derpprobe" "$derper"
     ''
+    + lib.optionalString stdenv.hostPlatform.isDarwin ''
+      wrapProgram $out/bin/tailscaled \
```

Comment:
Maybe add a comment explaining why this is needed?

Reply:
Sure thing, just pushed 4b2abf40c55821bada2664aff7474a77812a9bce